### ISSUE TITLE
Refreshing via F5 should not immediately refresh if in a Tetra League match

### ISSUE THREAD

**Issue by mat1jaczyyy** [o5k/tetrio-issues#756]
### Checklist (issue will be disregarded & closed if incomplete!)

- [x] I've **checked both [open and recently closed issues](https://github.com/tetrio/issues/issues?q=is%3Aissue)** to ensure this feature is not already requested.
- [x] I am using the **newest version** of TETR.IO.
- [x] I am certain this is a **feature request**, not a bug.
- [x] I've **thought this feature through** properly.

-----

### Is your feature request related to a problem? Please describe.
i pressed f5 thinking its on my firefox but actually tetrio desktop was focused and it made me lose a TL match i was winning 5-1 :(

### Describe the solution you'd like
pls f5 do nothing in TL unless i press it 5 times

### Describe alternatives you've considered
popup are you sure to refresh

### Additional context
:(

**Comment by A-GH-U** [comment]
Agreed, it is a very potentially punishing feature that has no safety nets to prevent accidental disconnections in matches!
